### DISCUSSION TITLE
Feature Request: Automated Backtesting for Multiple Stock Symbols

### DISCUSSION THREAD

**Discussion by Cosibility** [akumidv/tradingview-assistant-chrome-extension#198] [Ideas]
Dear Author,

Thank you for developing such a great tool like TradingView Assistant. I am very interested in it and have been using it for my strategy backtesting.

I would like to suggest a new feature for the tool: automated backtesting for multiple stock symbols. This feature would help users save time and automate the backtesting process for multiple symbols simultaneously.

I would greatly appreciate it if you could consider this and allow me to discuss further how this feature could be developed. Looking forward to hearing from you.

Thank you very much!

Best regards,
Thanh Hai

**Comment by akumidv** [comment]
Yes, this is in nearest plans, after finishing developing and deploying new major version

**Comment by Rubotimus** [comment]
Recently found your tool, its amazing, and i was looking for this exact feature aswell.
Are you open for donations to help the project a bit??
